source/features/tags-on-commits-list.tsx

```diff
@@ -35,6 +35,7 @@ function mergeTags(oldTags: CommitTags, newTags: CommitTags): CommitTags {
 	const result: CommitTags = {...oldTags};
 	for (const commit in newTags) {
 		if (result[commit]) {
+			// eslint-disable-next-line unicorn/prefer-spread
 			result[commit] = [...new Set(result[commit].concat(newTags[commit]))];
```

Comment:
Can't figure this one out

Reply:
I think this 
```suggestion
			// eslint-disable-next-line unicorn/prefer-spread
			result[commit] = [...new Set([...result[commit], ...newTags[commit]])];
```

Reply:
I promise I tried that!! Thanks

Reply:
If Sindre has a `concat-unique` package I’d use that. The code isn’t very readable here

Reply:
Of course he does: https://github.com/sindresorhus/array-union